evi check/Form1.cs

```diff
@@ -0,0 +1,650 @@
+            {
+                evidance.Add(evi2);
+            }
+            if (evi3 != null)
+            {
+                evidance.Add(evi3);
+            }
+
+            PosssibleOutcome();
+
+            if(evidance.Count() == 3)
+            {
+                if (evidance.Contains("EMF5") && (evidance.Contains("FingerPrints") && (evidance.Contains("Cold"))))
+                {
+                    this.output.Text = "Banshee";
+
+                }  else if ((evidance.Contains("GhostWriting")) && (evidance.Contains("SpiritBox") && (evidance.Contains("Cold"))))
+                {
+                    this.output.Text = "Demon";
+                }
+
+                else if ((evidance.Contains("EMF5")) && (evidance.Contains("SpiritBox") && (evidance.Contains("GhostOrb"))))
+                {
+                    this.output.Text = "Jinn";
+                }
+
+                else if ((evidance.Contains("SpiritBox")) && (evidance.Contains("Cold") && (evidance.Contains("GhostOrb"))))
+                {
+                    this.output.Text = "Mare";
+                }
+
+                else if ((evidance.Contains("EMF5")) && (evidance.Contains("GhostWriting") && (evidance.Contains("SpiritBox"))))
+                {
+                    this.output.Text = "oni";
+                }
+
+                else if ((evidance.Contains("EMF5")) && (evidance.Contains("Cold") && (evidance.Contains("GhostOrb"))))
+                {
+                    this.output.Text = "phantom";
+                }
+
+                else if ((evidance.Contains("FingerPrints")) && (evidance.Contains("SpiritBox") && (evidance.Contains("GhostOrb"))))
+                {
+                    this.output.Text = "poltergeist";
+                }
+
+                else if ((evidance.Contains("EMF5")) && (evidance.Contains("GhostWriting") && (evidance.Contains("FingerPrints"))))
+                {
+                    this.output.Text = "revenant";
+                }
+
+                else if ((evidance.Contains("EMF5")) && (evidance.Contains("GhostWriting") && (evidance.Contains("GhostOrb"))))
+                {
+                    this.output.Text = "shade";
+                }
+
+                else if ((evidance.Contains("GhostWriting")) && (evidance.Contains("FingerPrints") && (evidance.Contains("SpiritBox"))))
+                {
+                    this.output.Text = "spirit";
+                }
+
+                else if ((evidance.Contains("FingerPrints")) && (evidance.Contains("SpiritBox") && (evidance.Contains("Cold"))))
+                {
+                    this.output.Text = "wraith";
+                }
+
+                else if ((evidance.Contains("GhostWriting")) && (evidance.Contains("Cold") && (evidance.Contains("GhostOrb"))))
+                {
+                    this.output.Text = "yurei";
+                }
+                else
+                {
+                    this.output.Text = "Need more evidance";
+                }
```

Comment:
This block looks to be duplicating effort from the `monsterlist()` function. This could be rewritten to avoid the duplication, and create a single point of maintenance for adding a new monster
```suggestion
            foreach(KeyValuePair<string, List<string>> entry in ghostlist)
            {
                if (!evidence.Except(entry.value).Any())
                {
                    this.output.Text = entry.key;
                }
            }
```